frontend/src/Editor/Viewer.jsx

```diff
@@ -33,6 +33,7 @@ class Viewer extends React.Component {
       currentLayout: isMobileDevice ? 'mobile' : 'desktop',
       currentUser: authenticationService.currentUserValue,
       isLoading: true,
+      appLoading: true,
```

Comment:
what is the difference between `isLoading` and `appLoading`?

Reply:
`isLoading` if for the currentState. and `appLoading` is for the apps only